easybuild/easyconfigs/l/libX11/libX11-1.6.3-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'libX11'
+version = '1.6.3'
+versionsuffix = '-Python-2.7.9'
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """X11 client-side library"""
+
+toolchain = {'name': 'intel', 'version': '2015a'} 
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+builddependencies = [
```

Comment:
actually only in one of them, and we generally include build deps first, so I'll fix the other one